VCFv4.3.tex

```diff
@@ -564,23 +583,28 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 \end{samepage}
-This key can be derived from the REF/ALT fields but is useful for filtering.
-The reserved values must be used for the types listed below:
+SVTYPE is a primitive basic data type and must be specified for all structural variants.
+Its value can usually be deduced from the REF and ALT fields, however despite this redundancy it is quite useful for filtering.
```

Comment:
I actually think that the redundancy of SVTYPE does more harm than good... For one, it causes the definitions of structural variant types to diverge between two sections in the specification (symbolic alleles and SVTYPE). If I'm correct in presuming that the definitions are supposed to be the same, then could we maybe drop this list entirely and instead say something like “The list of acceptable SVTYPE values is exactly the same as the top-level structural variant types described in (link to section above)”?

Reply:
Outside of scope of this PR, will raise as a separate issue if necessary